docs/RELEASE-NOTES-3.7.9.1.md

```diff
@@ -63,8 +63,8 @@ where blocks (such as the delay block) would drop tags. Paul David stepped in
 with his first contribution to GNU Radio with a fix.
 
 Fix an issue with flat\_flowgraph calculating alignment by
-misinterpreting units of write and read indeces. This was only apparent when these
-indeces are non-zero (a flowgraph has stopped and restarted).
+misinterpreting units of write and read indices. This was only apparent when these
```

Comment:
We really shouldn't be messing with release logs even if they include errors. Then again, I don't care enough to make this an issue with this PR.